pysidt/decomposition.py

```diff
@@ -8,6 +8,8 @@ def atom_decomposition(mol):
 
 
 def atom_decomposition_noH(mol):
+    clear_decomposition(mol)
```

Comment:
Why is this necessary? Why is someone feeding in data with labels that we want the algorithm to ignore?

I having a little trouble thinking of an example where I think it's smart to do this...but one could imagine using this kind of predictor to predict rate coefficients for a given family what we would typically use a single evaluation tree for (a la reaction GAV) in which case the input data would have labels that are important for prediction...

We can certainly add checks that will throw errors if things don't look in order...but I'm not sure why we can't expect the user to provide a dataset with sensible labeling...